### ISSUE TITLE
cross-entropy for segmentation is failed

### ISSUE THREAD

**Issue by zacario-li** [zacario-li/Segmentation-based-deep-learning-approach-for-surface-defect-detection#2]
Use cross-entropy for segmentation network is failed, I'll fix it later

**Comment by zacario-li** [comment]
fixed, if you want to use ce for loss function, clamp your labels into (0,1) before put them into criterion.
